rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -2932,21 +2936,21 @@ error_code sys_fs_truncate2(ppu_thread&, u32 fd, u64 size)
 
 error_code sys_fs_get_mount_info_size(ppu_thread&, vm::ptr<u64> len)
 {
-	sys_fs.todo("sys_fs_get_mount_info_size(len=*0x%x)", len);
+	sys_fs.trace("sys_fs_get_mount_info_size(len=*0x%x)", len);
 
 	if (!len)
 	{
 		return CELL_EFAULT;
 	}
 
-	*len = 0x9;
+	*len = std::count_if(mp_array.begin(), mp_array.end(), [](lv2_fs_mount_point* mp){return mp != &g_mp_sys_no_device && mp != &g_mp_sys_dev_usb && mp->is_mounted;}) + std::popcount(g_mp_sys_dev_usb.is_mounted);
```

Comment:
This code is duplicated..
Use a function for this.

Reply:
Done.